Makefile

```diff
@@ -20,9 +20,14 @@ run: ## Run via godep (without building)
 	go run *.go
 
 all: test build docker ## Test, build and docker image build
+	
+fmt: ## Run go fmt for all files in the project (excluding vendor)
+	go fmt $(shell go list ./... | grep -v /vendor/)
 
-test: ## Perform both unit and integration tests
-	go test -v -cover -tags "unit integration" $(TEST_PACKAGES)
```

Comment:
ok fair enough, it is a clever idea. The problem is this is not Java lol , so this will force to the contributors developers that are used to program in `go` to do in a different way.